webpack.common.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+/* eslint-disable */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Not addressed